src/sysfiles.c

```diff
@@ -494,6 +505,105 @@ Int SyLoadModule( const Char * name, InitInfoFunc * func )
 
 #endif
 
+/****************************************************************************
+**
+*F * * * * * * * * * * finding location of executable * * * * * * * * * * * *
+*/
+
+/****************************************************************************
+** The function 'find_yourself' is based on code (C) 2015 Mark Whitis, under
+** the MIT License
```

Comment:
Googling for "Mark Whitis find_yourself" does not immediately reveal the source of this. So perhaps add an explicit URL lik http://stackoverflow.com/a/34271901/928031